code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm

```diff
@@ -238,6 +238,17 @@
 		if(dose == removed)
 			S.visible_message("<span class='warning'>[S]'s flesh sizzles where the water touches it!</span>", "<span class='danger'>Your flesh burns in the water!</span>")
 
+
+/datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(istype(M, /mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		S.adjustToxLoss(12 * removed) // A slime having water forced down its throat would cause much more damage then being splashed on it
+		if (!S.client && S.Target)
+			if(S.Target) // Like cats
```

Comment:
Since you're already doing `&& S.Target` in the above if, you can remove this and tab the lines below it back one.